hack/testing/test-upgrade.sh

```diff
@@ -196,7 +222,7 @@ addTriggers && \
 rebuildVersion "upgraded" && \
 
 upgrade "upgraded" && \
-verifyUpgrade true && \
+verifyUpgrade "upgraded" true && \
 
 ./e2e-test.sh $USE_CLUSTER && \
```

Comment:
My thought was we want each step to pass, otherwise we would kick out to:
```
oc get events -o yaml > $ARTIFACT_DIR/all-events.yaml 2>&1
exit 1
```
I can probably push the tests into their own functions to mirror what I'm trying to do, but I thought `-e` will exit the script if a step fails not just the function it's in.